static/app/views/performance/http/settings.ts

```diff
@@ -0,0 +1,10 @@
+import type {BadgeType} from 'sentry/components/featureBadge';
+
+export const RELEASE_LEVEL: BadgeType = 'alpha';
```

Comment:
type assertion will mean you don't need the cast

```suggestion
export const RELEASE_LEVEL = 'alpha' as BadgeType;
```

Reply:
True but it will lose type safety, because this becomes possible:

```jsx
export const RELEASE_LEVEL = 'alha' as BadgeType;
```

I'd rather have the type safety here, since this setting will be removed anyway